mem0/memory/main.py

```diff
@@ -170,6 +172,7 @@ def _add_to_vector_store(self, messages, metadata, filters):
             messages=[{"role": "user", "content": function_calling_prompt}],
             response_format={"type": "json_object"},
         )
+        new_memories_with_actions = parse_json_response(new_memories_with_actions)
```

Comment:
do you think we should still do "json.loads"?

Reply:
@prateekchhikara Hi. Thanks for reviewing.

I think it may be the name of the function that confused you.

The function `parse_json_response` only extracts the JSON content from the potential code block from the given origin response string, it doesn't call `json.loads()`, which means it doesn't return a dictionary but a string.

So we HAVE to call `json.loads()` after the parsing progress.